alpenhorn/update.py

```diff
@@ -64,12 +64,18 @@ def update_node(node):
 
     # Make sure this node is usable.
     if not node.mounted:
-        log.debug("Skipping unmounted node \"%s\"." % node.name)
+        log.debug("Skipping unmounted node \"%s\".", node.name)
         return
     if node.suspect:
-        log.debug("Skipping suspected node \"%s\"." % node.name)
+        log.debug("Skipping suspected node \"%s\".", node.name)
 
-    log.info("Updating node \"%s\"." % (node.name))
+    log.info("Updating node \"%s\".", node.name)
+
+    # Check if the node is acutally mounted in the filesystem
+    check_node = update_node_mounted(node)
+
+    if check_node is False:
```

Comment:
Isn't `if not check_node` considered more Pythonic? (And would continue to work if we changed update_node_mounted to return `None` instead of `False` on failure.)

Also, I'm waffling whether to recommend that `check_node` be renamed to "`is_mounted`" or just be replaced entirely with:
```py
if not update_node_mounted(node):
    return
```

Reply:
@jrs65 what do you think? Keep "`check_node`", rename to "`is_mounted`", or eliminate and use "`if not update_node_mounted(node)`" directly, since the variable is not used anywhere else in the method.